ajax/libs/logdown.js/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "logdown.js",
```

Comment:
Because you used npm auto-update, using npm package name might be a better idea?